src/EventListener/InvalidVersionNumberListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ApiProblemBundle\Event\CreateApiProblem;
+use Libero\ContentApiBundle\Exception\InvalidVersionNumber;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Translation\TranslatorInterface;
+
+final class InvalidVersionNumberListener
```

Comment:
seems like a lot of work to create a new listener, which will cover only one exception type. Could:

- have one listener take care of more than one exception type if they are closely related?
- provide a default or abstract listener in `api-problem-bundle` for projects to instantiate or extend?

Reply:
The fact that `api-problem.xml` has to configure all listeners points toward the first option (and more than one listener can still be created for custom behaviors)

Reply:
Yep (relates to https://github.com/libero/content-api-bundle/pull/4#discussion_r227715806 too).

Reply:
There's some differences in the translation parameters, eg https://github.com/libero/content-api-bundle/blob/79f4624d7241d7affb74db56b5139066e87ea0d7/src/EventListener/InvalidIdListener.php#L35-L37 

vs

https://github.com/libero/content-api-bundle/blob/79f4624d7241d7affb74db56b5139066e87ea0d7/src/EventListener/InvalidVersionNumberListener.php#L35-L37

so will look at the latter.

Reply:
Done in 433e671.